modules/nf-core/ampcombi/cluster/main.nf

```diff
@@ -0,0 +1,48 @@
+process AMPCOMBI_CLUSTER {
+    tag 'ampcombi'
+    label 'process_medium'
+
+    conda "${moduleDir}/environment.yml"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/ampcombi:0.2.2--pyhdfd78af_0':
+        'biocontainers/ampcombi:0.2.2--pyhdfd78af_0' }"
+
+    input:
+    path(summary_file)
```

Comment:
We dont need to identify the sample ID in both `ampcombi/complete` and `ampcombi/cluster`, because in teh original module it was devloped that it takes a list of files (the result from `ampcombi/parsetables`) and so when implemented in a pipeline all the files will be mixed in a channel and then input into the submodules as a list that doesnt take into account the sample ID.